scripts/agents/includes/reviewer-v2/tool-registry.js

```
@@ -0,0 +1,181 @@
+ * Load a tool module lazily
+ * @private
+ */
+function _loadTool(toolName) {
+  if (tools[toolName] !== null && tools[toolName] !== undefined) {
+    return tools[toolName];
+  }
+
+  try {
+    // Dynamically require tool modules
+    const toolPath = `./tools/${toolName}`;
+    tools[toolName] = require(toolPath);
```

ESLint

[error] 'require' is not defined.

(no-undef)